inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -614,11 +655,32 @@ void MKLDNNMVNNode::execute(mkldnn::stream strm) {
 
     Layout layout = getParentEdgeAt(0)->getDesc().getLayout();
 
-    auto src_data = reinterpret_cast<const float *>(srcMemPtr->GetData());
-    auto dst_data = reinterpret_cast<float *>(dstMemPtr->GetData());
-
     if (layout == C || layout == NC || layout == CHW || layout == NCHW || layout == NCDHW) {
-        mvn_pln(src_data, dst_data, getParentEdgeAt(0)->getDesc().getDims());
+        if (input_prec == Precision::FP32) {
```

Comment:
Same comments as for jit_softmax: 1) Tile processing inside JIT impl -> no template; 2) Refeference implementation only for FP32 and planar layout.

Reply:
This suggestion will be implemented in subsequent PRs.